scripts/set_gem_timing

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+NC_STRING="NC"
+SC_STRING="SC"
+# TODO:(josh) reflect on absolute paths
+IOC_PATH_NC="/reg/g/pcds/epics/ioc/fee/GasDetDAQ/R4.0.22"
+IOC_PATH_SC="/reg/g/pcds/epics/ioc/fee/GasDetDAQ/R4.0.22-NOTS"
+IOC_PATH=0
```

Comment:
What's the purpose of setting the variable here? It isn't used until it is overridden later on.

Reply:
Like the strings you see mainly as a mechanism to communicate to other devs.... thats probably somewhat ill principled and a comment would reflect that better

Reply:
We should probably start using the /cds/group instead of /reg/g
It'll be so hard to beat my muscle memory here...